superset-frontend/plugins/plugin-chart-echarts/src/MixedTimeseries/buildQuery.ts

```diff
@@ -46,6 +47,11 @@ export default function buildQuery(formData: QueryFormData) {
     ...formData,
   };
 
+  if (!!baseFormData.orderby) {
```

Comment:
I'm curious, do we really need the double bang here?
```suggestion
  if (baseFormData.orderby) {
```